utilities/ifix-migration-toolkit/src/main/java/org/egov/ifixmigrationtoolkit/models/FiscalEventRequest.java

```diff
@@ -0,0 +1,21 @@
+package org.egov.ifixmigrationtoolkit.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.*;
+import org.egov.common.contract.request.RequestHeader;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@ToString
+public class FiscalEventRequest {
+
+    @JsonProperty("requestHeader")
+    private RequestHeader requestHeader = null;
+
+    @JsonProperty("fiscalEvent")
+    private FiscalEvent fiscalEvent = null;
```

Comment:
Isn't this a list?

Reply:
mongodb fiscal event service has request with a single fiscalEvent field instead of a list.